akka23-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/akka/Env.scala

```diff
@@ -22,19 +34,21 @@ object Env extends com.typesafe.conductr.bundlelib.scala.Env {
    *
    * Also in the case where ConductR did start this bundle but there is no AKKA_REMOTE_OTHER_IPS value then the current
    * node is assumed to be starting the cluster.
+   *
+   * @param systemName the system name to use - requires that mkSystemName is used
+   * @return Typesafe Config that can be merged with any existing configuration - this config
+   *         declares Akka remoting and cluster concerns.
    */
-  def asConfig: Config = {
+  def asConfig(systemName: SystemName): Config = {
     val akkaRemoteEndpointName = sys.env.getOrElse("AKKA_REMOTE_ENDPOINT_NAME", "AKKA_REMOTE")
 
-    def presentSeedNode(protocol: String, bundleSystem: String, bundleSystemVersion: String, ip: String, port: String, n: Int): (String, String) =
-      s"akka.cluster.seed-nodes.$n" -> s"akka.$protocol://$bundleSystem-$bundleSystemVersion@$ip:$port"
+    def presentSeedNode(protocol: String, systemName: String, ip: String, port: String, n: Int): (String, String) =
+      s"akka.cluster.seed-nodes.$n" -> s"akka.$protocol://$systemName@$ip:$port"
```

Comment:
@huntc - In the case of the same bundle (i.e. same bundle name) with different `compatibilityVersion`, would this change cause these bundles to try to form the same cluster?

Also, with this change, is the user expected to provide different system name between different `compatibilityVersion`?